### PR TITLE
Milestone picker: show overdue milestones in light orange #1410

### LINKED ISSUE #1410: Milestone picker: show overdue milestones in light orange
We can show overdue ones in light orange.

### PR DESCRIPTION
**Pull request by garbanzos**
Fixes #1410.

Will continue to work on this after #1417 is merged.
 
image

### CODE REVIEW

File: src/main/java/ui/components/pickers/PickerMilestone.java [hunk 1]
```diff
@@ -78,7 +79,14 @@ private void adjustWidthToFont(Label milestone) {
...
 
     private void setOpenStatusColour(Label milestone) {
-        milestone.setStyle("-fx-background-color: " + (isOpen() ? OPEN_COLOUR : CLOSE_COLOUR) + ";");
+        String colour = OPEN_COLOUR;
+        if (!isOpen()) {
+            colour = CLOSE_COLOUR;
+        }
+        if (isOpen() && isOverdue()) {
+            colour = OVERDUE_COLOR;
+        }
```
Comment (damithc) [inline src/main/java/ui/components/pickers/PickerMilestone.java]: How about this?
```java 
String colour = isOpen() ? (isOverdue() ? OVERDUE_COLOR : OPEN_COLOUR) : CLOSED_COLOUR;
```

**Comment by garbanzos** [comment]
image

@damithc Is this orange ok?

**Comment by damithc** [comment]
yes, the orange is ok.

**Comment by garbanzos** [comment]
Ready for review!

**Comment by damithc** [comment]
@m133225 for your review.

File: src/main/java/ui/components/pickers/PickerMilestone.java [hunk 2]
```diff
@@ -78,7 +81,8 @@ private void adjustWidthToFont(Label milestone) {
     }
 
     private void setOpenStatusColour(Label milestone) {
```
Comment (m133225) [inline src/main/java/ui/components/pickers/PickerMilestone.java]: I think this method could be renamed to `setStatusColour`?

**Comment by m133225** [comment]
Just a small suggestion. Seems to be good otherwise.

**Comment by damithc** [comment]
Can merge after addressing comment.

**Pull request merged**